<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqPipelineBrowserContextMenu</class>
 <widget class="QWidget" name="pqPipelineBrowserContextMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>30</height>
   </rect>
  </property>
  <action name="actionPBChangeInput">
   <property name="text">
    <string>Change &amp;Input...</string>
   </property>
   <property name="iconText">
    <string>Change Input...</string>
   </property>
   <property name="toolTip">
    <string>Change a Filter's Input</string>
   </property>
   <property name="statusTip">
    <string>Change a Filter's Input</string>
   </property>
  </action>
  <action name="actionPBDelete">
   <property name="icon">
    <iconset>
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="statusTip">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionPBCopy">
   <property name="icon">
    <iconset>
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy Properties</string>
   </property>
  </action>
  <action name="actionPBPaste">
   <property name="icon">
    <iconset>
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="statusTip">
    <string>Paste Properties</string>
   </property>
  </action>
  <action name="actionPBIgnoreTime">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ignore Time</string>
   </property>
   <property name="toolTip">
    <string>Disregard this source/filter's time from animations</string>
   </property>
   <property name="statusTip">
    <string>Disregard this source/filter's time from animations</string>
   </property>
   <property name="whatsThis">
    <string>Disregard this source/filter's time from animations</string>
   </property>
  </action>
  <action name="actionPBOpen">
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqOpen24.png</normaloff>:/pqWidgets/Icons/pqOpen24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionPBCreateCustomFilter">
   <property name="text">
    <string>&amp;Create Custom Filter...</string>
   </property>
  </action>
  <addaction name="actionPBOpen"/>
  <addaction name="actionPBCopy"/>
  <addaction name="actionPBPaste"/>
  <addaction name="actionPBCreateCustomFilter"/>
  <addaction name="actionPBChangeInput"/>
  <addaction name="actionPBIgnoreTime"/>
  <addaction name="actionPBDelete"/>
 </widget>
 <resources>
  <include location="../../../Components/Resources/pqComponents.qrc"/>
  <include location="../../../Components/Resources/pqComponents.qrc"/>
 </resources>
 <connections/>
</ui>
